content/docs/esc/other-integrations/github-actions.md

```diff
@@ -0,0 +1,221 @@
+---
+title_tag: Integrate with GitHub Actions | Pulumi ESC
+title: GitHub Actions
+h1: "Pulumi ESC: Integrate with GitHub Actions"
+meta_desc: This page provides an overview of how to use Pulumi ESC with GitHub Actions.
+weight: 2
+menu:
+  pulumiesc:
+    parent: esc-other-integrations
+    identifier: esc-other-integrations-github-actions
+---
+
+## Overview
+
+Pulumi ESC integrates with [GitHub Actions](https://docs.github.com/en/actions) (GHA) to help developers automatically manage configuration and secrets when running workflows. Additionally, Pulumi ESC works with the [Pulumi CLI Action](https://github.com/marketplace/actions/pulumi-cli-action#pulumi-github-actions) to provide secrets to workflow jobs and steps, such as AWS Credentials.
+
+{{< notes type="info" >}}
+**What is Pulumi ESC?** Pulumi ESC (Environments, Secrets, and Configuration) allows you to define collections of configuration settings and secrets, known as environments, and utilize them in any application or service. [Learn more](https://www.pulumi.com/docs/esc/)
+{{< /notes >}}
+
+## Prerequisites
+
+Ensure you have:
+
+- installed [Pulumi ESC](https://www.pulumi.com/docs/install/esc/)
+- installed [GitHub CLI](https://cli.github.com/)
+- a Pulumi Cloud account. You can sign up for an [always-free, individual account](https://app.pulumi.com/signup).
+- a GitHub account. You can sign up for an [account here](https://github.com/join).
+
+{{< notes type="info" >}}
+**New to Pulumi ESC?** Complete the [Getting Started tutorial](https://www.pulumi.com/docs/esc/get-started/)
+{{< /notes >}}
+
+In addition to the Pulumi ESC and GHA integrations below, consider authenticating to your Pulumi Cloud with GitHub OIDC. [Follow the guide](https://www.pulumi.com/docs/pulumi-cloud/oidc/client/github/) to enable the [Pulumi Auth Action](https://github.com/marketplace/actions/pulumi-auth-action).
+
+## Log in to your GitHub account
+
+Programmatically log in to your GitHub account from your terminal without having to locally store your token. Note, that a fine-grained GitHub personal access token is required. Follow the [GitHub instructions](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token) to create one.
```

Comment:
Is this part a step? or a goal? Are we saying you need to log in ahead of time or the steps will show you how to programmatically log in?

Reply:
This is the goal itself. I'll clarify.

Reply:
Updated.